src/main/java/org/folio/helper/CheckinHelper.java

```diff
@@ -124,13 +124,9 @@ private ReceivingResults prepareResponseBody(CheckinCollection checkinCollection
       results.getReceivingResults().add(result);
 
       // Get all processed piece records for PO Line
-      Map<String, Piece> processedPiecesForPoLine = StreamEx
-        .of(piecesGroupedByPoLine.getOrDefault(poLineId, Collections.emptyList()))
-        .toMap(Piece::getId, piece -> piece);
+      Map<String, Piece> processedPiecesForPoLine = getProcessedPiecesForPoLine(poLineId, piecesGroupedByPoLine);
```

Comment:
some unused imports are introduced, you can find them in sonar report:
https://sonarcloud.io/project/issues?id=org.folio%3Amod-orders&pullRequest=936&resolved=false&sinceLeakPeriod=true